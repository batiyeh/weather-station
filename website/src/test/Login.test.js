import React from 'react'
import LoginForm from '../components/registration/loginForm'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
    .create(<LoginForm/>).toJSON();
    expect(tree).toMatchSnapshot();
})