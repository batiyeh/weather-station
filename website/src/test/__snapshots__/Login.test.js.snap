// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="login-container"
>
  <div
    id="login"
  >
    <img
      alt=""
      className="login-logo"
      height="200"
      src="space-satellite-dish-512x512.png"
      width="200"
    />
    <form
      action="/api/user/login/"
      id="loginForm"
      method="post"
    >
      <div
        className="login-info mb-3"
      >
        <div
          className="col-12 row no-padding-left"
        >
          <a
            className="forgot-link"
            href="/user/reset"
            id="forgot"
          >
            Forgot password?
          </a>
        </div>
        <div
          className="form-group"
        >
          <input
            className="form-control"
            id="username"
            name="username"
            placeholder="Username"
            type="text"
          />
        </div>
        <div
          className="form-group"
        >
          <input
            className="form-control"
            id="password"
            name="password"
            placeholder="Password"
            type="password"
          />
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-6 no-padding-left"
        >
          <a
            className="register-link"
            href="/user/create"
          >
            <button
              className="btn btn-default btn-block register-btn"
              type="button"
            >
              Register
            </button>
          </a>
        </div>
        <div
          className="col-6 no-padding-right"
        >
          <button
            className="btn btn-default btn-block login-btn"
            type="submit"
          >
            Login
          </button>
        </div>
      </div>
    </form>
  </div>
</div>
`;
